### PR TITLE
add KubeSail to pick-right-solution.md

### PR DESCRIPTION
**Pull request by pastudan**
KubeSail offers hosted Kubernetes, via a simple auth with GitHub

### CODE REVIEW

**Comment by gochist** [comment]
@pastudan Hi! I'm mainly working for Korean localization. Thank you for updating Korean also! But according to [the guideline](https://kubernetes.io/docs/contribute/intermediate/#working-with-localized-content), please separate this for each language. Or it is okay to make a single PR for English. Because Korean localization team periodically (by 2~3 weeks) tracks upstream(English) changes and apply them to downstream(Korean).

File: content/en/docs/setup/pick-right-solution.md [hunk 1]
```diff
@@ -75,6 +75,8 @@ cluster.
 
 * [Kublr](https://kublr.com) offers enterprise-grade secure, scalable, highly reliable Kubernetes clusters on AWS, Azure, GCP, and on-premise. It includes out-of-the-box backup and disaster recovery, multi-cluster centralized logging and monitoring, and built-in alerting.
 
+* [KubeSail](https://kubesail.com) is an easy, free way to try Kubernetes.
```
Comment (sftim) [inline content/en/docs/setup/pick-right-solution.md]: Free as in “tier”? I see there are [limits](https://kubesail.com/pricing).

Comment (pastudan) [reply content/en/docs/setup/pick-right-solution.md]: Hey thanks @sftim -- Yes, the free tier is limited, however what I'm trying to emphasize is that its a free tier with the lowest barrier to entry (a single click on the "Sign in with GitHub" button, and your namespace is created). All other free tiers that I'm aware of still require a credit card to get started.

In the spirit of keeping things simple, I trimmed down the wording as much as possible, but please feel free to revise the description to make it more clear.

**Comment by pastudan** [comment]
@gochist Thanks for clarifying! I removed the translation from this PR, and will leave it to the experts, as I do not speak any Korean :) Apologies for misunderstanding the process.

File: content/en/docs/setup/pick-right-solution.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/setup/pick-right-solution.md]: Seems like there genuinely is a free option here, so the wording you're proposed looks OK to me @pastudan

**Pull request merged**